docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,142 @@
 # Cody Ignore
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+<p className="subtitle">Learn how to ignore specific files or folders in your codebase when using Cody.</p>
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+The Cody Ignore feature helps you control what context from your codebase is used by Cody. You can do it via:
 
-## Enabling Unstable Features
+-   Cody Context Filters (Cody Enterprise Only)
+-   Cody Ignore File (All Cody users)
+
+## Cody Context Filters 
+<Callout type="info"> Cody Context Filters is available for Enterprise users and is currently in the Experimental stage.</Callout>
+Admins on the Sourcegraph Enterprise instance can use the **Cody Context Filters** to determine which repositories Cody can use as the context in its requests to third-party LLMs. This functionality is currently experimental and is only supported with the following requirements:
+
+- A valid Cody Enterprise license and a running Sourcegraph version `>=5.4.0`
+- Works if you're using Cody for VS Code, Cody for JetBrains, and Cody for Web as your client
+- Your Cody client meets the following version specs: VS Code `>=1.20.0` and JetBrains `>=6.0.0`
+- Setting the `cody-context-filters-enabled` feature flag to `true`
```

Comment:
```suggestion

```

I feel like we shouldn't even advertise this yet, the feature isn't ready. And it's worth noting that this is a feature flag on the server, not the client.

Reply:
@MaedahBatool let's talk through this during our 1:1 and @kalanchan's other point about whether these should be in the same page.